appdatabase/database.go

```diff
@@ -56,6 +57,13 @@ func InitializeDB(path, password string, kdfIterationsNumber int) (*sql.DB, erro
 	if err != nil {
 		return nil, err
 	}
+
+	// Migrate `settings.usernames` here, because current SQL implementaion doesn't support `json_each`
+	err = MigrateEnsUsernames(db)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
I think this should be moved to be executed before the `err = migrations.Migrate(db)` in line 56,  otherwise it will not be possible to execute migrations over the ens_usernames table in case we need to